Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM apache/airflow:2.6.0-python3.10
+FROM apache/airflow:2.8.3-python3.8
```

Comment:
can we use this image - `registry.cern.ch/cern-sis/airflow-base:2.8.3`? This is the base airflow image to be used across all the dags. It is airflow - 2.8.3 and python 3.10